### PR TITLE
Apply includes and excludes to GetById and GetByIds

### PR DESCRIPTION
**Pull request by ejsmith**

### CODE REVIEW

File: src/Foundatio.Repositories.Elasticsearch/Repositories/ElasticReadOnlyRepositoryBase.cs [hunk 1]
```diff
@@ -714,6 +720,79 @@ protected virtual ICommandOptions<T> ConfigureOptions(ICommandOptions<T> options
         return options;
     }
 
+    protected virtual void ConfigureGetRequest(GetRequest request, ICommandOptions options)
+    {
+        var (resolvedIncludes, resolvedExcludes) = GetResolvedIncludesAndExcludes(options);
+
+        if (resolvedIncludes.Length > 0 && resolvedExcludes.Length > 0)
+        {
```
Comment (niemyjski) [inline src/Foundatio.Repositories.Elasticsearch/Repositories/ElasticReadOnlyRepositoryBase.cs:728]: This seems odd to have both includes and excludes.

**Review by niemyjski — COMMENTED** [review]
Looks good, do you think we should hash includes and excludes and use it part of the cache key?

File: src/Foundatio.Repositories.Elasticsearch/Repositories/ElasticReadOnlyRepositoryBase.cs (hunk 1 above)
Comment (ejsmith) [reply src/Foundatio.Repositories.Elasticsearch/Repositories/ElasticReadOnlyRepositoryBase.cs:728]: You can 100% have both.

**Comment by ejsmith** [comment]
> Looks good, do you think we should hash includes and excludes and use it part of the cache key?

Seems like that is over-complicating things and we should do what you asked us to do.

**Review by niemyjski — APPROVED** [review]

**Pull request merged**